src/components/dashboard/dashboard.component.ts

```diff
@@ -98,6 +98,9 @@ export class DashboardComponent implements AfterViewInit, AfterContentInit, OnDe
     refreshLayout(): void {
         const didChangeLayout = this.dashboardService.shiftWidgetsUp();
 
+        // emit information about the layout
+        this.dashboardService.layout$.next(this.dashboardService.getLayoutData());
```

Comment:
I would perhaps suggest moving this down to just above the line where the `layoutChange` emit actually happens, because (correct me if I'm wrong) this should not be necessary if `didChangeLayout` is false.